demo-data/src/main/java/org/jboss/pnc/demo/data/DatabaseDataInitializer.java

```diff
@@ -168,7 +168,7 @@ public void initiliazeProjectProductData() {
                     .project(project1)
                     .description("Test build config for project newcastle")
                     .environment(environment1)
-                    .buildScript("mvn clean deploy -Dmaven.test.skip")
+                    .buildScript("mvn clean deploy -DskipTests=true")
```

Comment:
Timing a build on an internal server with skipTests versus maven.test.skip the difference appears fairly insignificant.